.vscode/settings.json

```diff
@@ -59,5 +59,6 @@
                 "Delete temporary files"
             ]
         },
-    ]
+    ],
+    "vorlage-latex.buildcontainer": "jemand771/latex-build:v1.0.0"
```

Comment:
die frage hier ist, ob wir das ganze image oder nur den tag rein schreiben wollen. im das image zu ändern müsste man sowieso in mehreren dateien rumschreiben (`.gitpod.yml` und `tasks.json` bzw. ab jetzt `settings.json`)
--> das image werden wir sicher nie ändern, von daher sollte das ja nicht so wild sein

Reply:
seh ich auch so.